coderd/database/queries/insights.sql

```diff
@@ -22,18 +22,18 @@ GROUP BY workspace_agent_stats.user_id, users.username, users.avatar_url
 ORDER BY user_id ASC;
 
 -- name: GetTemplateInsights :one
--- GetTemplateInsights has a granularity of 5 minutes where if a session/app was
--- in use, we will add 5 minutes to the total usage for that session (per user).
-WITH d AS (
-	-- Subtract 1 second from end_time to avoid including the next interval in the results.
-	SELECT generate_series(@start_time::timestamptz, (@end_time::timestamptz) - '1 second'::interval, '5 minute'::interval) AS d
-), ts AS (
+-- GetTemplateInsights has a granularity of 1 minute where if a session/app was
```

Comment:
Do we need to adjust site/UI according to this change?

Reply:
Do you mean the change from 5m -> 1m? Nope, this is just a change in how much "time spent in X" is calculated. This may reduce time spent slightly due to less over-attribution.